### ISSUE TITLE
Can't use certain /slack commands in threads

### ISSUE THREAD

**Issue by k4k** [wee-slack/wee-slack#477]
`/slack slash /shrug` in a thread produces the following error:

```
                        │15:47:08             weechat     | python: stdout/stderr: Traceback (most recent call last):
                        │15:47:08             weechat     | python: stdout/stderr:   File "/Users/twood/.weechat/python/autoload/wee_slack.py", line 112, in wrapper
                        │15:47:08             weechat     | python: stdout/stderr:     return f(*decode_from_utf8(args), **decode_from_utf8(kwargs))
                        │15:47:08             weechat     | python: stdout/stderr:   File "/Users/twood/.weechat/python/autoload/wee_slack.py", line 3079, in slack_command_cb
                        │15:47:08             weechat     | python: stdout/stderr:     EVENTROUTER.cmds[function_name]("", current_buffer, args)
                        │15:47:08             weechat     | python: stdout/stderr:   File "/Users/twood/.weechat/python/autoload/wee_slack.py", line 101, in wrapper
                        │15:47:08             weechat     | python: stdout/stderr:     return f(data, current_buffer, *args, **kwargs)
                        │15:47:08             weechat     | python: stdout/stderr:   File "/Users/twood/.weechat/python/autoload/wee_slack.py", line 3110, in command_slash
                        │15:47:08             weechat     | python: stdout/stderr:     team = channel.team
                        │15:47:08             weechat     | python: stdout/stderr: AttributeError: 'SlackThreadChannel' object has no attribute 'team'
                        │15:47:08             weechat =!= | python: error in function "slack_command_cb"
```

**Comment by alyssais** [comment]
This doesn't just affect `/slack` commands. I got a similar error attempting to `/query`:

```
Traceback (most recent call last):
  File "/Users/alyssa/.config/weechat/python/autoload/wee_slack.py", line 89, in wrapper       
    return f(data, current_buffer, *args, **kwargs)
  File "/Users/alyssa/.config/weechat/python/autoload/wee_slack.py", line 112, in wrapper      
    return f(*decode_from_utf8(args), **decode_from_utf8(kwargs))
  File "/Users/alyssa/.config/weechat/python/autoload/wee_slack.py", line 2989, in command_talk
    team = e.weechat_controller.buffers[current_buffer].team
AttributeError: 'SlackThreadChannel' object has no attribute 'team'
```

**Comment by trygveaa** [comment]
This has now been fixed (in commit 2e4ce42).
